src/main/java/org/fcrepo/transform/http/FedoraTransform.java

```diff
@@ -119,16 +122,27 @@ public void setUpRepositoryConfiguration() throws RepositoryException, IOExcepti
             // create the configuration base path
             jcrTools.findOrCreateNode(internalSession, "/fedora:system/fedora:transform", "fedora:Configuration",
                     "fedora:NodeTypeConfiguration");
-            final Node node =
-                jcrTools.findOrCreateNode(internalSession, CONFIGURATION_FOLDER + "default", NT_FOLDER, NT_FOLDER);
-            LOGGER.debug("Transforming node: {}", node.getPath());
-
-            // register an initial default program
-            if (!node.hasNode(NT_BASE)) {
-                final Node baseConfig = node.addNode(NT_BASE, NT_FILE);
-                jcrTools.uploadFile(internalSession, baseConfig.getPath(), getClass().getResourceAsStream(
-                        "/ldpath/default/nt_base_ldpath_program.txt"));
-            }
+
+            final Map<String, String> transformations = of(
+                    "default", "/ldpath/default/ldpath_program.txt",
+                    "deluxe", "/ldpath/deluxe/ldpath_program.txt");
+
+            transformations.entrySet().forEach(entry -> {
```

Comment:
Why not just `Map::forEach` here? You're just building up a set of `Entry` and then breaking it back down into keys and values?